docker.go

```diff
@@ -0,0 +1,44 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/loadimpact/k6/js/modules"
+)
+
+const version = "v0.0.2"
+
+func init() {
+	modules.Register("k6/x/docker", &Docker{
+		Version: version,
+	})
+}
+
+// Docker is the main export of k6 docker extension
+type Docker struct {
+	Version string
+}
+
+// Ps lists Docker containers
+func (d *Docker) Ps() []string {
```

Comment:
Should this be renamed to `ListContainers` or similar? I'm thinking that there might be no need to have matching terminology if it makes the module harder to read and use. 👍🏼

Reply:
✅